test.disabled/ejabberd_tests/tests/s2s_SUITE.erl

```diff
@@ -96,13 +96,13 @@ init_per_suite(Config0) ->
 
     Config1 = [{s2s_opts, S2S} | escalus:init_per_suite(Config0)],
     Config2 = [{escalus_user_db, xmpp} | Config1],
-    escalus:create_users(Config2, escalus:get_users([alice2, alice_bis, bob])).
+    escalus:create_users(Config2, escalus:get_users([alice2, alice, bob])).
```

Comment:
What is the reason behind these changes?

Reply:
Reverts s2s_SUITE back before 85cc37284fae81f25d14f71cfb3418f4a2402b83.
In that commit, alice was replaced with alice_bis, because it's was not possible to use alice in s2s.
I reverted it back, because we use alice everywhere else.